src/components/ArticlePreview.jsx

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'gatsby';
+
+import {
+  articlePreview,
+  rowSection,
+  authorSection,
+  infoSection,
+  imageLink,
+  dot,
+} from '../styles/articlePreview.module.scss';
+import Tag from './Tag';
+import ImageLink from './ImageLink';
+import ProfilePicture from './ProfilePicture';
+
+export const AuthorSection = ({ name, url, credentials, image }) => (
+  <div className={authorSection}>
+    {image && (
+      <ProfilePicture
+        url={url}
+        imageData={image.node.childImageSharp.gatsbyImageData}
+      />
+    )}
+    <Link to={url} aria-label={`Link to user profile: ${name}`}>
+      {credentials ? `${name},` : name}
+    </Link>
+    {credentials}
+  </div>
+);
+
+AuthorSection.propTypes = {
+  name: PropTypes.string.isRequired,
+  url: PropTypes.string.isRequired,
+  image: PropTypes.oneOfType([PropTypes.object, PropTypes.string]).isRequired,
+  credentials: PropTypes.string.isRequired,
+};
+
+const Dot = () => <div className={dot}>•</div>;
+
+const renderTags = (tags) => {
+  if (tags.length > 0) {
+    return tags.map((tag) => (
+      <div key={tag}>
+        <Tag>{tag}</Tag>
+        <Dot />
+      </div>
+    ));
+  }
+  return '';
+};
+
+const renderDate = (date) => {
+  const monthNames = [
+    'Jan',
+    'Feb',
+    'Mar',
+    'Apr',
+    'May',
+    'Jun',
+    'Jul',
+    'Aug',
+    'Sep',
+    'Oct',
+    'Nov',
+    'Dec',
+  ];
+  if (date) {
+    return (
+      <div>{`${date.getDate()} ${
+        monthNames[date.getMonth()]
+      }, ${date.getFullYear()}`}</div>
+    );
+  }
+  return '';
+};
+
+const renderReadingTime = (readingTime) => {
+  if (readingTime) {
+    return <div>{`${readingTime} read`}</div>;
+  }
+  return '';
+};
+
+const ArticlePreview = ({
+  children,
+  articleUrl,
+  previewImage,
+  tags,
+  title,
+  readingTime,
+  date,
+  authorName,
+  authorCredentials,
+  authorImage,
+  authorUrl,
+}) => (
+  <div className={articlePreview}>
```

Comment:
Here could maybe be an aria label somewhere, now I think it might go straight to the image 🤔